src/helm/benchmark/metrics/basic_metrics.py

```diff
@@ -840,6 +840,11 @@ def compute_calibration_metrics(per_instance_stats: Dict[Instance, List[Stat]])
             assert 0.0 <= cur_correct <= 1.0
             correct.append(int(cur_correct))
 
+    # If the model does not produce logprobs (i.e. max_prob is always 1.0)
+    # then don't compute calibration metrics.
+    if all([max_prob == 1.0 for max_prob in [max_probs]]):
```

Comment:
Why `[max_probs]`?  This looks odd...

Also, why does max_prob = 1.0 correspond to no logprobs?  Seems like you should still be able to compute calibration metrics?

Reply:
The extra `[` and `]` was a typo.

You can still compute calibration metrics, but it's the computed value is trivially (1 - accuracy) because basically we treat it as if the model is claiming 100% probability every time.

As discussed, I rewrote the check to use logprobs.